lib/test/index.html

```diff
@@ -1,21 +0,0 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <meta charset="utf-8">
-  <title>Ramda Tests</title>
-  <link rel="stylesheet" href="mocha.css" />
-</head>
-<body>
-  <h1>Tests for ramda</h1>
-  <div id="mocha"></div>
-  <script src="lib/test/expect.js"></script>
-  <script src="mocha.js"></script>
-  <script src="assert-shim.js"></script>
-  <!--[if lt IE 9]>
-    <script src="../lib/sauce/ie8-shim.js"></script>
-  <![endif]-->
-  <script>mocha.setup('bdd')</script>
-  <script src="bundle.js"></script>
-  <script src="../sauce/mocharun.js"></script>
-</body>
-</html>
```

Comment:
I've worked with a number of very junior JS developers over the years.  And even today, NPM is a mystery to many of them.  But a script that they can use via `<script src='ramda'></script>` is always familiar.  A test suite that runs as a web page is familiar.  Don't get me wrong.  I don't want to spend to much effort catering to them, but it would be great if we can support them without a lot of effort.  An included REPL sounds like a good idea.